packages/cli/test/integration/workflows/workflows.controller.ee.test.ts

```diff
@@ -428,28 +428,38 @@ describe('GET /workflows/:workflowId', () => {
 		expect(responseWorkflow.sharedWithProjects).toHaveLength(0);
 	});
 
-	test('should return workflow with credentials saying owner does not have access when not shared', async () => {
-		const savedCredential = await saveCredential(randomCredentialPayload(), { user: member });
+	test.each([
+		['owner', () => owner],
```

Comment:
For learning, why does this need to be a function? I'd've passed the actor directly.

Reply:
It's evaluated before the `beforeAll` block and thus it would just evaluate to `undefined` and pass this to `each`, whereas using a function creates a closure to the original binding and evaluates this only when called from inside the test. At that point `beforeAll` already ran.

I hope that makes sense.